packages/ui/src/components/Channel/index.tsx

```diff
@@ -278,7 +298,7 @@ export function Channel({
                           channel={channel}
                           group={group}
                           mode={headerMode}
-                          title={title}
+                          title={title ?? ''}
```

Comment:
you'll see a couple of diffs like this below – previously, when computing a title, we'd return `''` when we couldn't get a good title. I swapped this for `null` to be explicit, and coalesced to `''` where we're displaying the title.